rpm/harbour-cameraplus.spec

```diff
@@ -44,6 +44,14 @@ BuildRequires:  quill-qt5-utils
 %description
 Cameraplus is an advanced easy to use camera
 
+%package tools
+Summary: Tools for cameraplus
+Group: Applications/Multimedia
+Requires:  gst-droid = %{version}-%{release}
```

Comment:
This is not needed. You can require the main app but do not depend on gstreamer bits